rotkehlchen/chain/evm/decoding/ens/name_utils.py

```diff
@@ -0,0 +1,18 @@
+import re
+from typing import Final
+
+TLD_PATTERN: Final = re.compile(r'\.[a-z0-9-]{2,}$', re.IGNORECASE)
+
+
+def is_potential_ens_name(text: str) -> bool:
+    """Function used to tell if a string could be a potential ENS domain.
+    If .eth is found we exit fast but for other domains like .argent.xyz
+    we use a regex that detects valid TLDs. The developer has always to check
+    if indeed the address is valid or not.
+
+    The full list of TLDs supported by ENS is available at https://docs.ens.domains/dns/tlds
+    """
+    if text.endswith('.eth'):
+        return True
+
+    return bool(TLD_PATTERN.search(text))
```

Comment:
not worth the extra if check. The regex is simple and regex matching is fast.

```suggestion
    return bool(TLD_PATTERN.search(text))
```